testcafe/e2e.spec.js

```diff
@@ -0,0 +1,36 @@
+import { t } from 'testcafe'
+import { retryUntilResults, queryAPI, initPosthog, captureLogger, staticFilesMock, clearEvents } from './helpers'
+
+fixture('posthog.js capture')
+    .page('http://localhost:8000/playground/cypress/index.html')
+    .requestHooks(captureLogger, staticFilesMock)
+    .beforeEach(() => initPosthog())
+    .afterEach(async () => {
+        await clearEvents()
+
+        const browserLogs = await t.getBrowserConsoleMessages()
+        Object.keys(browserLogs).forEach((level) => {
+            browserLogs[level].forEach((line) => {
+                console.log(`Browser ${level}:`, line)
+            })
+        })
+
+        console.debug('Requests to posthog:', JSON.stringify(captureLogger.requests, null, 2))
+    })
+
+test('Captured events are accessible via /api/event', async (t) => {
+    await t
+        .click('[data-cy-custom-event-button]')
```

Comment:
Test and helpers look good. One thing I don't understand is where this attribute exists and how it's being clicked

Reply:
It comes from `playground/cypress/index.html` which was previously used by cypress tests as well.